File: ejercicios/src/main/java/co/edu/sena/ejercicios/ejercicio02/Anfibio.java
package co.edu.sena.ejercicios.ejercicio02;


public class Anfibio extends Vertebrado {
    @Override
    public void comer() {
        System.out.println("soy omnívoro");
    }

    @Override
    public void desplazarse() {
        System.out.println("camino y nado");
    }

    @Override
    public void reproducirse() {
        System.out.println("de forma sexual");
    }

    @Override
    public void nacer() {
        System.out.println("nazco de un huevo");
    }
}

File: typescript/src/ejercicio02/anfibio.ts
import {Vertebrados} from "./vertebrados";

export class Anfibio extends Vertebrados {

    public comer(): void {
        console.log("soy carnívoro");
    }

    public desplazarse(): void {
        console.log("nado, camino");
    }

    public reproducirse(): void {
        console.log("de forma sexual");
    }

    public nacer(): void {
        console.log("ovíparo");
    }
}
